### PR TITLE
Add KendoUI tooltip behavior

### PR DESCRIPTION
**Pull request by martin-g**

### CODE REVIEW

**Comment by martin-g** [comment]
I hope the formatting is OK :)

**Comment by sebfz1** [comment]
*Martin is on a killing spree!*

Many thanks for this addition, seems to be well formatted! :)

I really like the way you did (supplying the component as ctor parameter and the rendering is the model). I did it differently for jquery-ui tooltip (`CustomTooltipBehavior#newContent`). Maybe I can adapt it so I will have only one `TooltipBehavior` - instead of having TooltipBehavior and CustomTooltipBehavior - jquery-ui side...

File: wicket-jquery-ui-samples/src/main/java/com/googlecode/wicket/jquery/ui/samples/pages/kendo/tooltip/ComponentKendoTooltipPage.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		ImageAndTextPanel tooltip2 = new ImageAndTextPanel("imageAndText");
+		final Label imageAndTextLabel = new Label("imageAndText", "Hover me to give you a tip with an image!");
+		this.add(imageAndTextLabel);
+		imageAndTextLabel.add(new TooltipBehavior(tooltip2) {
+			@Override
+			public void onConfigure(Component component)
+			{
+				super.onConfigure(component);
+
+				getOptions().set("position", Options.asString("right"));
```
Comment (sebfz1) [inline wicket-jquery-ui-samples/src/main/java/com/googlecode/wicket/jquery/ui/samples/pages/kendo/tooltip/ComponentKendoTooltipPage.java]: fyi, you can use setOption instead...

Comment (martin-g) [inline wicket-jquery-ui-samples/src/main/java/com/googlecode/wicket/jquery/ui/samples/pages/kendo/tooltip/ComponentKendoTooltipPage.java]: Improved it!
Thanks!

**Comment by sebfz1** [comment]
Just ran the sample... I like the image, lol! :)

**Comment by martin-g** [comment]
:-)

**Pull request merged**